package.json

```diff
@@ -61,7 +61,7 @@
 		"prern-bundle": "patch-package --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
-		"bundle": "npm run clean; npm run bundle:js && npm run genstrings",
+		"bundle": "npm run clean; npm run core prebundle && npm run bundle:js && npm run genstrings",
```

Comment:
Good catch! I suppose the [pre scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts) we have in place (e.g. `prebundle`) were not correctly running because we many of our bundle scripts have suffixes (e.g. `:js`, `:android`) which causes the pre script to not execute.

Reply:
Yep, besides the `prebundle` command is defined in `packages/react-native-editor` and we usually run the `bundle` command on the root (GB-mobile).